apps/web/app/environments/[environmentId]/surveys/[surveyId]/edit/SurveyMenuBar.tsx

```diff
@@ -129,7 +129,7 @@ export default function SurveyMenuBar({
           className="w-72 border-white hover:border-slate-200 "
         />
       </div>
-      {localSurvey?.responseRate && (
+      {!!localSurvey?.responseRate&& (
```

Comment:
Have you tested this?

I'm sceptical this "&&" operator works without the space.

And why did you invest this?

Reply:
@jobenjada This is a file That I didn't touch. Not sure, how this got pushed. Will invert this